org.eclipse.lyo.oslc4j.core.plugin/category.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<site>
+   <feature url="features/org.eclipse.lyo.oslc4j.client.plugin.feature_4.0.0.qualifier.jar" id="org.eclipse.lyo.oslc4j.client.plugin.feature" version="4.0.0.qualifier">
```

Comment:
it there a way not to check this file in or not to have 4.0.0 identifier in the file?

Reply:
What file are you referring to?

Reply:
category.xml here says "feature_4.0.0.qualifier.jar". this will have to be updated all the time, right?

Reply:
That's on my todo list (See description of the pull request). I Want to avoid, but don't know how yet.

Reply:
@berezovskyi this issue is now solved. No more references to the versions.